lib/index.js

```diff
@@ -737,3 +735,15 @@ api.consensus._scheduleConsensusWork = (job, callback) => {
     callback();
   }
 };
+
+function _use(plugin, callback) {
+  {
+    let p;
+    try {
+      p = injector.use(plugin);
+    } catch(e) {
+      return callback(e);
+    }
+    callback(null, p);
+  }
```

Comment:
Extra `{}`?

Reply:
TIL that extra curly braces is still valid syntax.  Fixed, ty.